pkgs/by-name/cr/cryptpad/package.nix

```diff
@@ -59,6 +61,17 @@ let
     }
   ];
 
+  x2t_version = "v7.3+1";
```

Comment:
(gmrbl github doesn't allow commenting outside of context)

^ v7 a few lines above should probably b updated to e1267803 as well (current version as of 2024.9.1)

And we'll need to add a test that checks these versions are in sync probably, it's too easy to not notice :/

Reply:
I'll implement this in a followup PR!

Reply:
Done in https://github.com/NixOS/nixpkgs/pull/365514